pycbc/population/__init__.py

```diff
@@ -1,2 +1,5 @@
 from pycbc.population.rates_functions import *
 from pycbc.population.scale_injections import *
+from pycbc.population.fgmc_functions import *
+from pycbc.population.fgmc_laguerre import *
+from pycbc.population.fgmc_plots import *
```

Comment:
I might skip this import here, as importing matplotlib things at the top level can sometimes cause unforeseen problems (i.e. conflicts with other matplotlib settings that depend on who calls first)

Reply:
Sounds reasonable.